jekyll/_cci2/ip-ranges.md

```diff
@@ -175,8 +178,9 @@ CircleCI *does not recommend* configuring an IP-based firewall based on the AWS
 **IP ranges** is the recommended method for configuring an IP-based firewall to allow traffic from CircleCI’s platform.
 
 ## Known limitations
-{: #knownlimiations}
+{: #knownlimitations}
 
 - There currently is no support for specifying IP ranges config syntax when using the [pipeline parameters feature](https://circleci.com/docs/2.0/pipeline-variables/#pipeline-parameters-in-configuration).  Details in this [Discuss post](https://discuss.circleci.com/t/ip-ranges-open-preview/40864/6).
 - IP ranges is currently available exclusively for the [Docker executor](https://circleci.com/docs/2.0/executor-types/#using-docker), not including `remote_docker`.
 - If your job enables IP ranges and _pushes_ anything to a destination that is hosted by the content delivery network (CDN) [Fastly](https://www.fastly.com/), the outgoing job traffic **will not** be routed through one of the well-defined IP addresses listed above. Instead, the IP address will be one that [AWS uses](https://circleci.com/docs/2.0/ip-ranges/#awsandgcpipaddresses) in the us-east-1 or us-east-2 regions. This is a known issue between AWS and Fastly that CircleCI is working to resolve.
+- When downloading/uploading files from/to a destination hosted by the CDN [Cloudflare](cloudflare.com), intermittently, the job may receive TCP reset (RST) packets and drop the connection.  This could cause the job to fail if there is no retry logic in place while downloading the file.  CircleCI recommends including robust retry mechanisms in your config when attempting to download large files during execution of jobs with IP ranges enabled.  For example, if your job uses curl to download a large file, include the "--retry <num>" flag and set "<num>" to a large number such as 1,000."
```

Comment:
```suggestion
- When downloading/uploading files from/to a destination hosted by the CDN [Cloudflare](cloudflare.com), intermittently, the job may receive TCP reset (RST) packets and drop the connection. This could cause the job to fail if there is no retry logic in place while downloading the file. CircleCI recommends including robust retry mechanisms in your config when attempting to download large files during execution of jobs with IP ranges enabled. For example, if your job uses curl to download a large file, include the "--retry <num>" flag and set "<num>" to a large number such as 1,000."
```